apps/emqx_rule_engine/src/emqx_rule_actions.erl

```diff
@@ -204,7 +203,7 @@ increase_and_publish(ActId, Msg) ->
     emqx_metrics:inc_msg(Msg).
 
 -spec on_action_create_do_nothing(action_instance_id(), Params :: map())
-      -> NewParams :: map().
+      -> {bindings(), NewParams :: map()}.
```

Comment:
NIT: I have seen a lot of places having `->`  leading the line, elvis will probably not rock.
what kind of convention are we following ?

Reply:
No conventions for that, I'll follow the guide of elvis.